repos/system_upgrade/el7toel8/actors/sssdcheck/actor.py

```diff
@@ -0,0 +1,62 @@
+from leapp.actors import Actor
+from leapp.libraries.common.reporting import report_generic, report_with_remediation
+from leapp.models import SSSDConfig
+from leapp.reporting import Report
+from leapp.tags import IPUWorkflowTag, ChecksPhaseTag
+
+
+class SSSDCheck(Actor):
+    '''
+    Check SSSD configuration for changes in RHEL8 and report them.
+
+    These changes are:
+    - id_provider=local is no longer supported and will be ignored
+    - ldap_groups_use_matching_rule_in_chain was removed and will be ignored
+    - ldap_initgroups_use_matching_rule_in_chain was removed and will be ignored
+    - ldap_sudo_include_regexp changed default from true to false
+    '''
+
+    name = 'sssd_check'
+    consumes = (SSSDConfig,)
+    produces = (Report,)
+    tags = (IPUWorkflowTag, ChecksPhaseTag)
+
+    def process(self):
+        model = next(self.consume(SSSDConfig))
```

Comment:
What, if there's no message SSSDConfig?
```suggestion
        model = next(self.consume(SSSDConfig), SSSDConfig(domains=[]))
```

Reply:
Thank you. I used the latter one.